feat: add staging build commands

mritense opened the pull request:
Also includes some prettier line ending fixes,

bddjong commented:
To fix the cross-platform prettier issue, you'll need to find a way to deal with Git converting line endings to CRLF for windows users. For the component library, we fixed this through the following git attributes file: https://github.com/nl-design-system/denhaag/blob/master/.gitattributes

More information: https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf

mritense commented:
> To fix the cross-platform prettier issue, you'll need to find a way to deal with Git converting line endings to CRLF for windows users. For the component library, we fixed this through the following git attributes file: https://github.com/nl-design-system/denhaag/blob/master/.gitattributes
> 
> More information: https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf

I now added a line to the eslint config in newest commit (top comment: https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-delete-cr-prettier-prettier). Do you reckon this .gitattributes edit is still necessary?

bddjong commented:
> > To fix the cross-platform prettier issue, you'll need to find a way to deal with Git converting line endings to CRLF for windows users. For the component library, we fixed this through the following git attributes file: https://github.com/nl-design-system/denhaag/blob/master/.gitattributes
> > More information: https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf
> 
> I now added a line to the eslint config in newest commit (top comment: https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-delete-cr-prettier-prettier). Do you reckon this .gitattributes edit is still necessary?

I think it could in theory lead to inconsistent line endings across files, but I don't think there's any actual impact from that so nice fix!
